docs/_pages/tutorials/read/how-to-use.md

```diff
@@ -9,7 +9,12 @@ sidebar:
   title: '<img id="side-icon-verybig" src="/assets/vector/chalkboard-user.svg"/>How to use'
   nav: tutorial
 ---
+## How to Use (v0.3.3)
+![image](https://user-images.githubusercontent.com/97640728/218118681-520e7fa6-52e9-4740-b488-1d294f833de6.png)
+### <a href="https://www.athena-alpha.com/how-to-buy-bitcoins-using-robosats/" target="_blank">How To Buy Bitcoins Using RoboSats: Fastest KYC Free Sats In The West</a>
+Updated for the latest version, Athena Alpha's full guide walks you through the full buying process step-by-step. It also answers any extra FAQs you might have and helps outline how brand new Bitcoin Beginners can get the deposit needed to make their first trade.
```

Comment:
Can we delete "Updated for the latest version"? It only makes sense if the review/tutorial will keep up to date. However, we are planning v0.3.4 release as early as tomorrow so it would not be the latest version anymore :D

Reply:
Removed :)